lib/beaker-hostgenerator/data.rb

```diff
@@ -42,23 +41,12 @@ def pe_dir(version)
         ''
       end
 
-      gem_version = Gem::Version.new($1)
-      if(gem_version < Gem::Version.new("#{MASTER_PE_VERSION}") || version =~ /#{base_regex}-rc\d+\Z/)
-        pe_branch = $1
-      elsif gem_version < Gem::Version.new("#{MAIN_PE_VERSION}")
-        #Is this a Master PEZ build?
-        if(version =~ /.*(PEZ|pez)_.*/)
```

Comment:
I don't really understand why we were doing this in the first place, when we were already adding "feature" in the `source` switch statement above.  It ended up doing `master/feature/feature/ci-ready`, which was always wrong.